server/src/CompletionProvider.ts

```diff
@@ -63,54 +59,50 @@ export class CompletionProvider {
     return insertString
   }
 
-  createCompletionItem (keyword: string): CompletionItem[] {
-    let completionItem: CompletionItem[] = new Array < CompletionItem >()
+  createCompletionItemForDirectiveStatementKeyword (keyword: string): CompletionItem[] {
+    let completionItem: CompletionItem[] = []
 
     switch (keyword) {
       case 'inherit':
-        completionItem = this.convertElementInfoListToCompletionItemList(
-          this._projectScanner.classes,
-          this._classCompletionItemKind
-        )
+        completionItem = [
+          ...this.convertElementInfoListToCompletionItemList(
+            this._projectScanner.classes,
+            this._classCompletionItemKind
+          )]
         break
 
       case 'require':
       case 'include':
-        completionItem = completionItem.concat(
-          this.convertElementInfoListToCompletionItemList(
+        completionItem = [
+          ...this.convertElementInfoListToCompletionItemList(
             this._projectScanner.includes,
```

Comment:
Completion items are great for this!
However as stated in my comment, not only .inc files can be included.
Example: https://github.com/Freescale/meta-freescale-distro/blob/7956a0ab407a33c40fdc6eb4fabdcb7dc54fd359/conf/distro/include/fslc-base.inc#L1
But since these may be from other layers it's groing to be a pain. The best practice is to use .inc files. We can keep this IMO.